### ISSUE TITLE
Add tags for the generated content of the command gen-doc

### ISSUE THREAD

**Issue by romain-gsr** [mesg-foundation/core#507]
Command **gen-doc** is great but it overwrites any existing **README.md**.

Could you add surrounding tags for the generated content of the command gen-doc ?
This way the content can be seamlessly updated in a specific part of the README.

**Comment by Roms1383** [comment]
As a side note I would add that the currently generated header containing :

> # my-project

should be optional because maybe people might want to generate their own (with shields badges for examples, it's quite trendy) while avoiding duplicating the title.

**Comment by antho1404** [comment]
Good point, maybe the title should be set only if there is no readme or the readme is empty

**Comment by krhubert** [comment]
@mesg-foundation/core todo: close or move to [cli issues](https://github.com/mesg-foundation/cli/issues)
